### ISSUE TITLE
SAML Login with chunk-<id>.js files stuck pending status

### ISSUE THREAD

**Issue by damonb123** [apache/cloudstack#8194]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
UI, SAML
~~~

##### CLOUDSTACK VERSION

~~~
4.18.1.0
~~~

##### SUMMARY

Initial login of user to CloudStack hangs when choosing SAML.  The user appears to be logged in but screen is white and indicates it is still loading.

##### STEPS TO REPRODUCE

~~~
This issue started after upgrading from 4.17.2.0 to 4.18.1.0.

Initial login to CloudStack using SAML URL=https://<ULR>/client/index.html#/user/login?redirect=/
After choosing SAML login, CloudStack redirects to URL=https://<ULR>/client/index.html#/dashboard

Watching browser network screen, a number of chunk-<id>.js files show "Pending" status.
If you open a new tab and use "https://<URL>/client/index.html", the user is redirected to "https://<URL>/client/index.html#/dashboard" with no issues.

The original page will still be stuck on blank screen indicating the page is still loading. 
~~~

##### EXPECTED RESULTS

~~~
Dashboard page would be presented to user.
~~~

**Comment by DaanHoogland** [comment]
could this possibly be a cache problem, @damonb123 ?

**Comment by damonb123** [comment]
Yes, tested after clearning cache, and also tried incognito windows with same results.
This only occurs when using SAML.  LDAP login is not affected.

**Comment by rohityadavcloud** [comment]
@damonb123 can you check
- if the file exists on the server, are you using CloudStack management server to serve the UI or using a reverse proxy (and using the cloudstack-ui package)?
- if you're using the cloudstack-management package, can you check if the specific `chunk-<id>.js` exists on /usr/share/cloudstack-management/webapp/js path.
- is this stock UI or were there changes to the UI, or UI was rebuilt and copied to the management (or reverse proxy) server?

**Comment by rohityadavcloud** [comment]
I checked 4.18.1.0 based QA env which has SAML enabled - I haven't face the problem. I suspect it could either be a build or acl/access issue.
On https://qa.cloudstack.cloud/simulator/#/user/login?redirect=/ I used SAML to login as `rick` user and didn't face this, this env uses 4.18.1.0 (but doesn't install pkgs from upstream packages repo).
